ccpp/data/GFS_typedefs.meta

```diff
@@ -716,6 +730,30 @@
   dimensions = (horizontal_loop_extent)
   type = real
   kind = kind_phys
+[dust12m_in]
+  standard_name = fengsha_dust12m_input
+  long_name = fengsha dust input
+  units = various
+  dimensions = (horizontal_dimension,12,5)
+  type = real
+  kind = kind_phys
+  active = (flag_for_rrfs_smoke_coupling)
+[emi_in]
+  standard_name = anthropogenic_background_input
```

Comment:
background what? emissions? If so, how about `anthropogenic_background_emissions`?

Reply:
Yes, the emi_data.nc file only contains anthropogenic background emissions, so that name is fine.